markdown/preface.markdown

```diff
@@ -102,29 +115,29 @@ In short, SciPy and related libraries will be driving much of scientific data an
 > -- http://www.scipy.org/
 
 The SciPy ecosystem is a loosely defined collection of Python packages.
-In Elegant SciPy we will see many of its main players [^learn]:
+In Elegant SciPy we will meet many of its main players:
 
 * **NumPy** is the foundation of scientific computing in Python. It
 provides efficient numeric arrays and  wide support for numerical computation, including linear algebra, random numbers, and Fourier transforms.
-NumPy's killer feature are its "N-dimensional arrays", or `ndarray`.
+NumPy's killer feature is its "N-dimensional arrays", or `ndarray`.
 These data structures store numeric values efficiently and define a grid in any number of dimensions (more about this later).
 http://www.numpy.org/
 * **SciPy**, the library,
-is a collection of efficient, user-friendly numerical algorithms.
-It contains toolboxes for specific domains such as signal processing, integration, optimization, and statistics.
+is a collection of efficient numerical algorithms for domains such as signal processing, integration, optimization, and statistics.
+These are wrapped in user-friendly interfaces.
 http://www.scipy.org/scipylib/index.html
 * **Matplotlib**
-is a powerful package for plotting in two dimensions (and basic 3D). It draws its name from the syntax that it shares with Matlab.
+is a powerful package for plotting in two dimensions (and basic 3D). It draws its name from its Matlab-inspired syntax.
 http://matplotlib.org/
 * **IPython**
-is an interactive interface for Python, so you can quickly interact with your data and test ideas.
-* **Jupyter**
-The Jupyter notebook runs in your browser and allows you to write code in line with text and mathematical expressions, displaying the results of computation within the text.
+is an interactive interface for Python, which allows you to quickly interact with your data and test ideas.
+* The **Jupyter**
+notebook runs in your browser and allows you to write code in line with text and mathematical expressions, displaying the results of computation within the text.
 This entire book has been written with Jupyter.
 Jupyter started out as an IPython extension, but now supports multiple languages, including Cython, Julia, R, Octave, Bash, Perl and Ruby.
 http://jupyter.org
 * **pandas**
-provides fast, easy-to-use data structures, particularly to work with labelled data sets such as tables or relational databases, and manage time series.
+provides fast data structures in an easy-to-use package, particularly to work with labelled data sets such as tables or relational databases, and manage time series.
```

Comment:
data structures -> mixed data structures, columner data structures, or something of the sort to indicate that it's different from numpy in this regard.